{
  "uid" : "73c0328e2da48405",
  "name" : "[US-111]/[1] I can log into automationpractice.com using registered email \"thor.odinson@example.com\" & password \"12345\"",
  "fullName" : "tests.WebAPP.Authentication_Tests.test_1",
  "historyId" : "f5964073fccecb7bcdd82f60f0557540",
  "time" : {
    "start" : 1602694597659,
    "stop" : 1602694603665,
    "duration" : 6006
  },
  "description" : "[US-111]/[1] As a user I can log into automationpractice.com using registered email \"thor.odinson@example.com\" & password \"12345\"",
  "descriptionHtml" : "<p>[US-111]/[1] As a user I can log into automationpractice.com using registered email &quot;thor.odinson@example.com&quot; &amp; password &quot;12345&quot;</p>\n",
  "status" : "failed",
  "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
  "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_1(Authentication_Tests.java:43)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
  "flaky" : false,
  "newFailed" : true,
  "beforeStages" : [ {
    "name" : "Setting up Excel File",
    "time" : {
      "start" : 1602694523548,
      "stop" : 1602694523599,
      "duration" : 51
    },
    "description" : "Setting up Excel File",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Setting up Test Class",
    "time" : {
      "start" : 1602694596704,
      "stop" : 1602694597654,
      "duration" : 950
    },
    "description" : "Setting up Test Class",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "[US-111]/[1] As a user I can log into automationpractice.com using registered email \"thor.odinson@example.com\" & password \"12345\"",
    "status" : "failed",
    "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
    "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_1(Authentication_Tests.java:43)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
    "steps" : [ {
      "name" : "I open home page",
      "time" : {
        "start" : 1602694602715,
        "stop" : 1602694602848,
        "duration" : 133
      },
      "status" : "failed",
      "statusMessage" : "Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]",
      "statusTrace" : "java.lang.AssertionError: Ups, something went really bad! Even Michael Scofield couldn't have predicted that error! :) expected [http://automationpractice.com/index.php] but found [http://automationpractice.com/]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.steps.MainPageSteps.iOpenHomePage(MainPageSteps.java:35)\n\tat tests.WebAPP.Authentication_Tests.test_1(Authentication_Tests.java:43)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "88f4813dee5121fd",
      "name" : "Scenario FAIL screenshot",
      "source" : "88f4813dee5121fd.png",
      "type" : "image/png",
      "size" : 18343
    }, {
      "uid" : "c088f43dce0b4ddb",
      "name" : "TestNG test FAIL logs",
      "source" : "c088f43dce0b4ddb.txt",
      "type" : "text/plain",
      "size" : 2722
    } ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "Teardown Test Class",
    "time" : {
      "start" : 1602694603667,
      "stop" : 1602694603735,
      "duration" : 68
    },
    "description" : "Teardown Test Class",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Teardown Test Suite",
    "time" : {
      "start" : 1602695039259,
      "stop" : 1602695040517,
      "duration" : 1258
    },
    "description" : "Teardown Test Suite",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "package",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "testClass",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "testMethod",
    "value" : "test_1"
  }, {
    "name" : "parentSuite",
    "value" : "Automationpractice.com WEB - Kamil Nowocin"
  }, {
    "name" : "suite",
    "value" : "Automated tests for WEB client"
  }, {
    "name" : "subSuite",
    "value" : "tests.WebAPP.Authentication_Tests"
  }, {
    "name" : "host",
    "value" : "travis-job-a8c8b075-3079-49fb-9f08-6d508dcf90e2"
  }, {
    "name" : "thread",
    "value" : "6396@travis-job-a8c8b075-3079-49fb-9f08-6d508dcf90e2.TestNG-test=Automated tests for WEB client-4(26)"
  }, {
    "name" : "framework",
    "value" : "testng"
  }, {
    "name" : "language",
    "value" : "java"
  }, {
    "name" : "story",
    "value" : "POSITIVE FLOW"
  }, {
    "name" : "owner",
    "value" : "Kamil Nowocin"
  }, {
    "name" : "epic",
    "value" : "Web App Tests"
  }, {
    "name" : "feature",
    "value" : "LOGIN TESTS"
  }, {
    "name" : "severity",
    "value" : "critical"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ {
    "name" : "TAP-0001",
    "url" : "https://tracker.FAKE.com/jira/browse/AUTOMATION_PRACTICE/issue/TAP-0001",
    "type" : "issue"
  }, {
    "name" : "STORY-111",
    "url" : "https://tracker.FAKE.com/jira/browse/AUTOMATION_PRACTICE/STORY-111",
    "type" : "tms"
  } ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "critical",
    "owner" : "Kamil Nowocin",
    "retries" : [ {
      "uid" : "c0a303fbe7ee8064",
      "status" : "skipped",
      "statusDetails" : "Expected condition failed: waiting for element to be clickable: Proxy element for: DefaultElementLocator 'By.xpath: //a[@class='login']' (tried for 15 second(s) with 500 milliseconds interval)",
      "time" : {
        "start" : 1602694571323,
        "stop" : 1602694596639,
        "duration" : 25316
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 5,
        "skipped" : 14,
        "passed" : 278,
        "unknown" : 0,
        "total" : 300
      },
      "items" : [ {
        "uid" : "6bf766a7f408b85f",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189766563/#testresult/6bf766a7f408b85f",
        "status" : "passed",
        "time" : {
          "start" : 1602608202822,
          "stop" : 1602608220931,
          "duration" : 18109
        }
      }, {
        "uid" : "10b5eea1448335d0",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189536094/#testresult/10b5eea1448335d0",
        "status" : "passed",
        "time" : {
          "start" : 1602521830028,
          "stop" : 1602521851170,
          "duration" : 21142
        }
      }, {
        "uid" : "c206e749aaae8b45",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189323165/#testresult/c206e749aaae8b45",
        "status" : "passed",
        "time" : {
          "start" : 1602435348798,
          "stop" : 1602435367168,
          "duration" : 18370
        }
      }, {
        "uid" : "a83d6cfec881875a",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189207237/#testresult/a83d6cfec881875a",
        "status" : "passed",
        "time" : {
          "start" : 1602348923609,
          "stop" : 1602348944118,
          "duration" : 20509
        }
      }, {
        "uid" : "a7a267c087a45ae7",
        "reportUrl" : "https://travis-ci.com/kamil-nowocin/Test_Automation-automationpractice/builds/189083485/#testresult/a7a267c087a45ae7",
        "status" : "passed",
        "time" : {
          "start" : 1602262493612,
          "stop" : 1602262513158,
          "duration" : 19546
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "73c0328e2da48405.json",
  "parameterValues" : [ ]
}